index.html

```diff
@@ -134,19 +134,19 @@ <h2 class="textBox__head textBox__head--md textBox__head--white">
             </div>
           </article>
           <article class="content content--vertical content--mobile-mb-3">
-            <div class="imageBox imageBox--half">
+            <div class="imageBox">
               <img
                 class="imageBox__image imageBox__image--bLine"
                 src="./img/salad.png"
                 alt="salad in a bowl and parsley on the side"
               />
             </div>
             <div class="textBox">
-              <h2 class="textBox__head textBox__head--md textBox__head--white">
+              <h2 class="textBox__head textBox__head--md textBox__head--grey">
                 Taking Taste Further
               </h2>
               <p
-                class="textBox__desc textBox__desc--sm textBox__desc--grey textBox__desc--maxL-3"
+                class="textBox__desc textBox__desc--sm textBox__desc--lightgrey textBox__desc--maxL-3"
```

Comment:
Are you going to switch to bootstrap? The main benefit is that I don't need to understand your CSS naming strategy. And you CSS code will be more feature focused. You can do this in another pull request. Just want to know your plan.

Reply:
Yes I'm going to.
The purpose of this pull request is to know what I can get from invision, so I can go forward without hesitation.

I won't merge this PR :)